### ISSUE TITLE
item.localTemporaryPath.path Undefined

### ISSUE THREAD

**Issue by asoufi** [jibon57/nativescript-mediafilepicker#109]
This plugin crashes as soon as I click DONE after selecting 1 or more image/video. It also crashes if I click DONE without selecting anything.

I am using the provided example. The code compiles and deploys ok. I am using NativeScript 6.5.1 and iOS 13.3.1

The error message is:

JavaScript error:
file: node_modules/nativescript-mediafilepicker/mediafilepicker.ios.js:67:0 JS ERROR TypeError: undefined is not an object (evaluating 'item.localTemporaryPath.path')

Any idea/suggestion on how to fix this error?

**Comment by ParthMaisheri** [comment]
Facing the same issue.

**Comment by asoufi** [comment]
After spending several hours debugging it, it turned out there was nothing wrong with the plugin itself, the issues was NativeScript version. I thought it was using the latest (6.5.0) but it was not. After removing node_modules and making sure package.json had the correct NativeScript version, I ran "npm install" and the provided example worked as expected.

**Comment by ParthMaisheri** [comment]
@asoufi i tried the above steps but didn't help me ,
@jibon57 can you please help.

**Comment by DJilanov** [comment]
I have the same issue

**Comment by asoufi** [comment]
Just make sure you are using the latest versions of NativeScript and your plugins. There was nothing else I did to get the plugin to work. As a matter of fact, I used the same example provided in its documentation. While trying to debug the issue, I decided to create a simple project that uses the plugin only. When it worked, I determine the issue was not the plugin but my code. As I reviewed my code, I discovered that I was not using the latest version of NS. As soon as I updated it, the plugin and its example worked.
